lib/ui/search_ol_screen/search_ol_screen.dart.dart

```diff
@@ -40,6 +40,7 @@ class _SearchOLScreenState extends State<SearchOLScreen>
     required String title,
     String? subtitle,
     required String author,
+    String? description,
```

Comment:
I think this is not necessary as description will be downloaded on the AddBookScreen.